datumaro/plugins/cityscapes_format.py

```diff
@@ -335,11 +342,15 @@ def apply(self):
         self.save_label_map()
 
     def save_label_map(self):
-        path = osp.join(self._save_dir, CityscapesPath.LABELMAP_FILE)
-        labels = self._extractor.categories()[AnnotationType.label]
-        if len(self._label_map) > len(labels):
-            self._label_map.pop('background')
-        write_label_map(path, self._label_map)
+        if self._save_dataset_meta:
+            save_meta_file(self._save_dir, self._extractor.categories())
```

Comment:
Maybe we should make it a class method?

Reply:
Do you mean add this function to the `Converter` class?

Reply:
Yes, exactly. Then, it can be overridden, if necessary.

Reply:
Done.